tracks/ruby/exercises/series/mentoring.md

```diff
@@ -0,0 +1,49 @@
+New concepts: Iterating over an array (preferably with `each_cons`, avoid `each_with_index`); `.chars` to split the string; (private) `attr_reader`; raising an Argument Error; inline if-statement; `unless` 
+
+### Minimal solution for approval
+
+```ruby
+
+class Series
+
+  def initialize(numerals)
+    @numerals = numerals.chars
+  end
+
+  def slices(span)
+    raise ArgumentError unless span <= numerals.size
+    numeral.each_cons(span).map(|slice| slice.join )
```

Comment:
I believe this should have an `s` after `numeral`

Reply:
Thanks! I opened a new PR to fix it. #698